.ci/build.sh

```diff
@@ -6,6 +6,11 @@ mkdir private _site public
 
 EXPORTED_DATA="static/tasks.yaml static/instances.yaml"
 
+EXTERNAL_REPO="gh-board"
```

Comment:
We dont need this variable.  It is only used once.

And we will have other external repos in the future.